hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1060,6 +1061,41 @@ func (r *HostedClusterReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		}
 	}
 
+	// Reconcile External CCM once kubeconfig is available
+	if hcp.Status.KubeConfig != nil {
+		var ccmImage, ccmComponentName string
```

Comment:
Why define `ccmImage` here if it is only used inside the `ccmComponentImage` condition?